bcgov/biohubbc-platform
# SIMSBIOHUB-25: Admin dashboard and  project roll up

# Overview

## Links to Jira tickets

[SIMSBIOHUB-25](https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-25)

## Description of relevant changes

- Added admin dashboard to biohub
- Added new endpoint to serve data to admin dashboard
- Updated routing to account for admin only access
- Migration: updated source transform to include new fields
- Migration: added new columns for tracking submitter system and elastic search object

## Testing Procedure
1. Get both sims and biohub running locally
2. Open biohub and login
3. You should be redirected to the new dashboard page
4. See there are `No Pending Security Reviews`
5. Create a project and survey in SIMS
6. Add documents to both project and survey (track the number of files in total)
7. Refresh the dashboard page (this might take a bit as the transform action is queued)
8. See the data table is updated with the project you just created. Compare the Files to Review with the total files you added to both Project and Survey (they should be the same)
9. Click the name of the project and be redirected to the project details page

## PR Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### Code


### Style


### Documentation


### Tests


### Linting/Formatting

       _See the `lint` commands in package.json_
       _See the `format` commands in package.json_

### SonarCloud
